k9mail/src/main/res/values/strings.xml

```diff
@@ -1203,6 +1203,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="recipient_error_non_ascii">Special characters are currently not supported!</string>
     <string name="recipient_error_parse_failed">Error parsing address!</string>
     <string name="account_settings_crypto_support_sign_only">Support signing of unencrypted messages</string>
+    <string name="account_settings_crypto_support_default_opportunistic">Try to encrypt every message</string>
```

Comment:
First, I thought a positive preference that is checked by default with the label "Encrypt by default" is good but then I looked at the current crypto preference screen.
The first two prefs: "OpenPGP app" and "My Key" are most important and the third one is an advanced preference disabled by default: "support signing of unencrypted messages". Thus, I would use "Disable automatic encryption" as a second advanced preference that is also disabled by default. This way all advanced preferences are disabled by default, which makes sense.

Maybe even group them together and rename the second one to make it shorter:
* OpenPGP app  
* My Key  
* Advanced (section title, similar to titles in Global Settings -> Display)  
* * [ ] Disable automatic encryption
* *  [ ] Support unencrypted signing